examples/acquiring_data/downloading_cutouts.py

```diff
@@ -41,7 +41,7 @@
 # See `this page <http://jsoc.stanford.edu/ajax/register_email.html>`_
 # for more details.
 
-jsoc_email = os.environ["JSOC_EMAIL"]
+jsoc_email = "<your email address>"
```

Comment:
Did you push in-between commits?

Reply:
yes but i ammended all commits to my previous one with no edit.

Reply:
I see but this change is now incorrect (the old code has to stay) and it doesn't change the other two examples you did from before.

Reply:
i undo all changes except one line which is why only one change is showing up. All other changes were removed.

Reply:
Ah I see but this is now incorrect. We need to tell readers of the the examples that they need to replace this line with their email address and make it really clear to prevent any misunderstanding.

Reply:
what do you suggest i do next. shall i remove all changes and sync with origin and add another commit with only the line changing?

Reply:
I am not sure I follow. We need to restore this line back to normal and add a comment above each of these lines in each example telling people what to change if they want to use their own email address.

Reply:
gotcha. I'll make these changes right now. So i can remove all the os.environ["JSOC_EMAIL"] and instead tell users to edit the example files themselves right? Also they will be replaced with placeholder string for their email addresses.

Reply:
gotcha! Thanks

Reply:
lmk if the latest commit addresses your comments.